modules/prometheus-exporter_oracle/conf/readme.yaml

```diff
@@ -0,0 +1,3 @@
+documentations:
+
+source_doc:
```

Comment:
add requirements and installation process of the exporter / agent to make the metrics collection works